### ISSUE TITLE
Pattern in StructureDefinition not reflected in Instance

### ISSUE THREAD

**Issue by markkramerus** [FHIR/sushi#148]
mCODE's `CancerGenomicsReport` profile has a fixed code (implemented via a pattern) on `category`.  There is an instance example of this class named `mCODECancerGenomicsReportExample01`. The instance doesn't have the category it should have inherited from its structure definition it is an `InstanceOf`. This results in an (accurate) IG Pub error:
`Profile http://hl7.org/fhir/us/mcode/StructureDefinition/CancerGenomicsReport, Element 'DiagnosticReport.category': minimum required = 1, but only found 0`

**Comment by cmoesel** [comment]
I looked into this.  The fixed category is coming from the USCore profile `us-core-diagnosticreport-lab`:
<img width="746" alt="image" src="https://user-images.githubusercontent.com/2278253/73125239-a1b47000-3f72-11ea-83c8-71a11070dc5a.png">

You can see there that category is sliced and the slicename is `LaboratorySlice`.  When we run SUSHI on fsh-mcode, one of the errors we get is:
```
error: Cannot resolve element from path: category:LaboratorySlice
```
I think this is related.

So, this looks like it is actually the same as #121.  We'll be working on #121 first thing this coming week.

**Comment by cmoesel** [comment]
This may have been fixed in 0.6.3, but is definitely fixed in 0.7.0.
